test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -0,0 +1,46 @@
+package seedu.addressbook.commands;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.testutil.TypicalTestPersons;
+
+/**
+ * Test class for the Find Command
+ */
+public class FindCommandTest extends CommandTest {
+
+    @Test
+    public void execute_find_isCaseSensitive() throws IllegalValueException {
```

Comment:
This name does not follow the convention. For example, the 3rd part of the name should be the expected result.